src/behavior-not-considered-unsafe.md

```diff
@@ -37,3 +37,18 @@ See [RFC 560] for error conditions, rationale, and more details about
 integer overflow.
 
 [RFC 560]: https://github.com/rust-lang/rfcs/blob/master/text/0560-integer-overflow.md
+
+##### Logic errors
+
+Some safe functionality may impose extra logic constraints that cannot be
+checked at neither compile-time nor runtime, for different reasons. If a user
```

Comment:
```suggestion
checked at neither compile-time nor runtime, for different reasons. If a program
```

Programs, not users, break constraints.

```suggestion
checked at neither compile-time nor runtime. If a user
```

"for different reasons" doesn't actually mean anything.

Reply:
> "for different reasons" doesn't actually mean anything.

Yeah, it is a bit informal for the reference. Initially I was thinking that a reader might want to know what kind of things wouldn't be possible to enforce, so I thought about giving some examples, and then reduced it... but yeah, it looks out of place.

Reply:
FWIW, this entire page is basically non-normative. It exists mainly as a reminder to the user that `safe` doesn't actually mean safe in general.

Reply:
Ah, thanks for the pointer, I didn't know that. I thought everything was more or less normative (well, "equally normative", since the intro says the document isn't normative anyway).